app/src/main/java/com/fish/monsters/features/history/ui/components/HistoryItem.kt

```diff
@@ -0,0 +1,164 @@
+import androidx.compose.ui.draw.drawWithContent
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.DpSize
+import androidx.compose.ui.unit.dp
+import com.fish.monsters.R
+import com.fish.monsters.common.extensions.drawNeonStroke
+import com.fish.monsters.common.extensions.isPreview
+import com.fish.monsters.common.utils.settings.SettingsManager
+import com.fish.monsters.common.views.PreviewContainer
+import com.fish.monsters.common.views.buttons.OutlinedFishButton
+import com.fish.monsters.core.database.entities.Contest
+import com.fish.monsters.core.database.entities.contest.Award
+import com.fish.monsters.core.theme.DarkPrimaryColor
+import com.fish.monsters.core.database.entities.contest.DifficultyLevel
+import com.fish.monsters.core.database.entities.contest.Duration
+import com.fish.monsters.core.database.entities.contest.GameLocation
+import org.koin.compose.koinInject
+
+@Composable
+fun HistoryItem(
+    contestInfo: Contest,
+    neonStyle: Boolean = if (isPreview()) false else koinInject<SettingsManager>().state.value.neonStyles,
+) {
+    OutlinedFishButton(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(66.dp)
+            .drawWithContent {
+                drawContent()
+                if (neonStyle)
+                    drawNeonStroke(DpSize(12.dp, 31.dp), DarkPrimaryColor)
+            },
+        onClick = { println(contestInfo) },
+        indentationSize = DpSize(12.dp, 31.dp),
+        contentPadding = PaddingValues(13.dp, 14.dp),
+    ) {
+        Row(Modifier.fillMaxSize()) {
+            Column(Modifier.width(200.dp)) {
+                Row(
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    Icon(
+                        imageVector = Icons.Default.CalendarMonth,
+                        contentDescription = "Calendar Icon",
+                        tint = DarkPrimaryColor,
+                        modifier = Modifier.width(16.dp)
+                    )
+                    Text(text = contestInfo.date, Modifier.padding(3.dp, 0.dp))
+                }
+                Spacer(
+                    modifier = Modifier
+                        .fillMaxHeight()
+                        .weight(1f)
+                )
+                Row {
+                    val difficultyStringRes = when (contestInfo.difficultyLevel) {
+                        DifficultyLevel.LOW -> R.string.low_difficulty
+                        DifficultyLevel.MEDIUM -> R.string.medium_difficulty
+                        DifficultyLevel.HIGH -> R.string.high_difficulty
+                    }
+                    Text(
+                        text = stringResource(id = difficultyStringRes),
+                        color = contestInfo.difficultyLevel.color
+                    )
+                }
+            }
```

Comment:
`Spacer(Modifier.weight(1f))` tutaj pozwoli ci na automatyczne rozsunięcie tych dwóch kolumn, możessz tez zastosować `horizontalArrangement = Arrangement.SpaceBetween` w Row powyżej. to pozwoli ci też uniknąć statycznych szerokości tych kolumn.